### DISCUSSION TITLE
Pterodacty Login

### DISCUSSION THREAD

**Discussion by MauRoblesss** [Paymenter/Paymenter#377] [Ideas]
My idea is something simple that I don't think will be very complicated to implement.

I consider it somewhat uncomfortable for buyers or users to receive an email to create their password in the pterodactyl panel, since this must be explained to the user because not everyone expects this. I propose that when purchasing a server, your password for the panel is the same. which is used in paymenter, or some type of integration to log into the panel from payment directly, because the truth is that it is a bit annoying for the user not to know where they receive the panel credentials, at least in my case the clients always They are asking where the server reaches them,

That would be my idea.

**Comment by CorwinDev** [comment]
We can't implement that the password of Paymenter is the same as Pterodactyl, but we might be able to add SSO (single sign on) to Pterodactyl/Paymenter

**Reply by lucasprds** [reply]
Hi ! SSO would be great awesome !

**Comment by Moreau741** [comment]
It is also generally not safe to set exactly the same password automatically as Pterodactyl, the customer will get an email immediately (when ordering) where they can set their own password (if mail servers are set properly)

**Comment by permahaze** [comment]
Is this still planned? (Pterodactyl SSO officially without third-party extension)

**Reply by CorwinDev** [reply]
If you sponsor us you get access to Pterodactyl SSO (9/month plan)

**Comment by krtk-2-rare** [comment]
Is it under development to add SSO
